LuckySpin.Test/SpinServiceTest.cs

```diff
@@ -17,16 +17,17 @@ public void SpinService_CalculateAvgWins_WinningSpin()
             var mockRepo = new Mock<ISpinRepository>();
             //TODO: Use the Setup() and Returns() methods of mockRepo
             //       to arrange for a consistent, expected output based on TestData
-
+            mockRepo.Setup(s => s.GetSpins()).Returns(SpinListData.GetSpins());
+            mockRepo.Setup(c => c.GetCount()).Returns(SpinListData.GetCount());
             var service = new SpinService(mockRepo.Object);
 
             //Act - run the method that you are testing and get a result
             double result = service.CalculateAvgWins();
 
             //Assert - compare the expected output from TestData to the method result
             // TODO: check the repo data for the number of previous spins and wins, add one winning spin
-                double wins = 1/*???*/, count=1/*???*/;
-                double expected = wins / count;
+            double wins = 4 + 1, count = SpinListData.GetCount() + 1;
+            double expected = wins / count;
             Assert.Equal(expected, result);
         }
```

Comment:
Also include the test for 
`public void SpinService_CalculateAvgWins_LosingSpin() {  /* mostly the same code as above, but different expected value */ }`